<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTorrent</class>
 <widget class="QDialog" name="AddTorrent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add torrent</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Torrent</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Torrent:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="TorrentFile_"/>
        </item>
        <item>
         <widget class="QPushButton" name="TorrentBrowse_">
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Files:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTreeView" name="FilesView_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="MarkMenuButton_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Mark...</string>
            </property>
            <property name="icon">
             <iconset theme="checkbox"/>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExpandAll_">
            <property name="text">
             <string>Expand all</string>
            </property>
            <property name="icon">
             <iconset theme="view-list-tree"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CollapseAll_">
            <property name="text">
             <string>Collapse all</string>
            </property>
            <property name="icon">
             <iconset theme="view-list-text"/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="Destination_"/>
          </item>
          <item>
           <widget class="QPushButton" name="DestinationBrowse_">
            <property name="text">
             <string>Browse...</string>
            </property>
            <property name="icon">
             <iconset theme="document-open-folder"/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="AvailSpaceLabel_">
            <property name="text">
             <string>unknown</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="AvailSpaceBar_">
            <property name="value">
             <number>0</number>
            </property>
            <property name="format">
             <string>%p% used</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tags:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LC::Util::TagsLineEdit" name="TagsEdit_">
        <property name="text">
         <string>untagged</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Add as:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="AddTypeBox_">
        <item>
         <property name="text">
          <string>Started</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paused</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="TryLive_">
        <property name="text">
         <string>Try to play this torrent live</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QFormLayout" name="InfoLayout_">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="TrackerURLLabel_">
        <property name="text">
         <string>Tracker URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="TrackerURL_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Size_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="CreatorLabel_">
        <property name="text">
         <string>Creator:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="Creator_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="CommentLabel_">
        <property name="text">
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Comment_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="DateLabel_">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="Date_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="MarkAll_">
   <property name="text">
    <string>Mark all</string>
   </property>
  </action>
  <action name="UnmarkAll_">
   <property name="text">
    <string>Unmark all</string>
   </property>
  </action>
  <action name="MarkSelected_">
   <property name="text">
    <string>Mark selected</string>
   </property>
  </action>
  <action name="UnmarkSelected_">
   <property name="text">
    <string>Unmark selected</string>
   </property>
  </action>
  <action name="MarkExisting_">
   <property name="text">
    <string>Mark existing files</string>
   </property>
  </action>
  <action name="MarkMissing_">
   <property name="text">
    <string>Mark missing files</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC::Util::TagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">util/tags/tagslineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>OK_</sender>
   <signal>released()</signal>
   <receiver>AddTorrent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel_</sender>
   <signal>released()</signal>
   <receiver>AddTorrent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TorrentFile_</sender>
   <signal>returnPressed()</signal>
   <receiver>OK_</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Destination_</sender>
   <signal>returnPressed()</signal>
   <receiver>OK_</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TagsEdit_</sender>
   <signal>returnPressed()</signal>
   <receiver>AddTorrent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
